src/engine/NativeFunctionCall.ts

```diff
@@ -428,34 +426,30 @@ export class NativeFunctionCall extends InkObject {
 
       // String operations
       this.AddStringBinaryOp(this.Add, (x, y) => x + y); // concat
-      this.AddStringBinaryOp(this.Equal, (x, y) => (x === y ? 1 : 0));
-      this.AddStringBinaryOp(this.NotEquals, (x, y) => (!(x === y) ? 1 : 0));
-      this.AddStringBinaryOp(this.Has, (x, y) => (x.includes(y) ? 1 : 0));
-      this.AddStringBinaryOp(this.Hasnt, (x, y) => (x.includes(y) ? 0 : 1));
+      this.AddStringBinaryOp(this.Equal, (x, y) => x === y);
+      this.AddStringBinaryOp(this.NotEquals, (x, y) => !(x === y));
+      this.AddStringBinaryOp(this.Has, (x, y) => x.includes(y));
+      this.AddStringBinaryOp(this.Hasnt, (x, y) => x.includes(y));
 
       this.AddListBinaryOp(this.Add, (x, y) => x.Union(y));
       this.AddListBinaryOp(this.Subtract, (x, y) => x.Without(y));
-      this.AddListBinaryOp(this.Has, (x, y) => (x.Contains(y) ? 1 : 0));
-      this.AddListBinaryOp(this.Hasnt, (x, y) => (x.Contains(y) ? 0 : 1));
+      this.AddListBinaryOp(this.Has, (x, y) => x.Contains(y));
+      this.AddListBinaryOp(this.Hasnt, (x, y) => x.Contains(y));
```

Comment:
I think we're missing a `!` here, eg. `!x.Contains(y)`

Reply:
Wow, great catch! That would have been annoying to debug down the line. I'll fix that!